package-overrides/github/fengyuanchen/cropperjs@0.7.0.json

```diff
@@ -0,0 +1,16 @@
+{
+  "main": "dist/cropper",
+  "registry": "jspm",
+  "dependencies": {
+    "css": "*"
+  },
+  "shim": {
+    "dist/cropper": [
+      "./cropper.css!"
+    ]
+  },
+  "files": [
+    "dist/cropper.js",
+    "dist/cropper.css"
+  ]
```

Comment:
It can be advisable to leave this out for the npm package directory as it contains so few files. It can be nice to have the original scss folder for some users.